### PR TITLE
Support BeaconContent#deviceMessage (dm)

### PR DESCRIPTION
**Pull request by kazuki-ma**

### CODE REVIEW

File: line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContent.java [hunk 1]
```diff
@@ -32,10 +32,27 @@
...
+     *
+     * <p>The beacon.dm property is only included in webhooks from devices that support the “device message” property.
+     * If device message is not included in webhooks, value is {@code null}.</p>
+     *
+     * <p>You can use beacon.dm with the LINE Simple Beacon specification.</p>
+     *
+     * @see <a href="https://github.com/line/line-simple-beacon/blob/master/README.en.md#line-simple-beacon-frame">LINE Simple Beacon specification (en)</a>
+     * @see <a href="https://github.com/line/line-simple-beacon/blob/master/README.ja.md#line-simple-beacon-frame">LINE Simple Beacon specification (ja)</a>
+     */
+    private final String dm;
```
Comment (yappo) [inline line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContent.java]: Would you like to add `getBeaconMessage()` method?

File: line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContentUtil.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+package com.linecorp.bot.model.event.beacon;
+
+import javax.xml.bind.DatatypeConverter;
+
+import lombok.experimental.PackagePrivate;
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
+@PackagePrivate
```
Comment (tokuhirom) [inline line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContentUtil.java]: I don't suggest to use lombok.experimental.* classes.

**Comment by kazuki-ma** [comment]
@tokuhirom Thank you. updated~

**Comment by yohsuke** [comment]
LGTM 👍

File: line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContent.java [hunk 3]
```diff
@@ -32,10 +32,56 @@
...
+     * Device message of beacon that was detected in byte[] format. (Optional)
+     *
+     * <p>The “device message” consists of data generated by the beacon to send notifications to bots.
+     *
+     * <p>The beacon.dm property is only included in webhooks from devices that support the “device message” property.
+     * If device message is not included in webhooks, value is {@code null}.</p>
+     *
+     * <p>You can use beacon.dm with the LINE Simple Beacon specification.</p>
+     *
+     * @see #getDeviceMessageAsHex() ()
```
Comment (yohsuke) [inline line-bot-model/src/main/java/com/linecorp/bot/model/event/beacon/BeaconContent.java]: duplicated `()` ?

**Comment by kazuki-ma** [comment]
🙇

**Pull request merged**